build.sh

```diff
@@ -349,6 +349,10 @@ if [ ${BUILD_BE} -eq 1 ] ; then
         CXX_COMPILER_LAUNCHER=${CCACHE}
     fi
 
+    if [[ -z ${WITH_STARCACHE} ]]; then
```

Comment:
make it a command line option and show it in the param summary output.

Reply:
> make it a command line option and show it in the param summary output.

done